src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -44,6 +44,12 @@ const printName = name => name.slice(10, -6) // removes 'notebooks/' and the .ip
 
 const noWrite = 'You do not have access to modify this workspace.'
 
+const sortOptions = [
+  { label: 'Name: Ascending', value: { field: 'name', direction: 'asc' } },
+  { label: 'Name: Descending', value: { field: 'name', direction: 'desc' } },
+  { label: 'Updated: Descending', value: { field: 'updated', direction: 'desc' } }
```

Comment:
Is this just newest first? If so, let's call it that.

Reply:
fixed